core/src/test/scala/eu/shiftforward/apso/json/JsonConvertSpec.scala

```diff
@@ -1,22 +1,43 @@
 package eu.shiftforward.apso.json
 
-import spray.json._
-import org.specs2.mutable.Specification
 import scala.collection.JavaConverters._
 
-class JsonConvertSpec extends Specification {
+import io.circe.Json
+import io.circe.parser.decode
+import org.specs2.matcher.EitherMatchers
+import org.specs2.mutable.Specification
+import spray.json._
+
+class JsonConvertSpec extends Specification with EitherMatchers {
   "JsonConvert" should {
-    "be able to convert scala objects to JSON" in {
-      JsonConvert.toJson(Map("a" -> 2, "b" -> Map(3 -> 7))) mustEqual """{ "a": 2, "b": { "3": 7 }}""".parseJson
-      JsonConvert.toJson(2) mustEqual "2".parseJson
-      JsonConvert.toJson("2") mustEqual """"2"""".parseJson
-      JsonConvert.toJson(2.5) mustEqual "2.5".parseJson
-      JsonConvert.toJson(List(1, 2, 3, 4)) mustEqual """[1, 2, 3, 4]""".parseJson
+    "be able to produce Spray Json models" in {
+      "by converting scala objects to JSON" in {
+        JsonConvert.toCirceJson(Map("a" -> 2, "b" -> Map(3 -> 7))) mustEqual decode[Json]("""{ "a": 2, "b": { "3": 7 }}""").right.get
```

Comment:
You can just use `parse` from the `io.circe.parser` package object and avoid the extra decoding step to `Json`.

Reply:
Simplified in d8e1b4f